src/rez/solver.py

```diff
@@ -2083,15 +2143,15 @@ def resolved_ephemerals(self):
         final_phase = self.phase_stack[-1]
         return final_phase._get_solved_ephemerals()
 
-    def reset(self):
+    def reset(self) -> None:
         """Reset the solver, removing any current solve."""
         if not self.request_list.conflict:
-            phase = _ResolvePhase(self.request_list.requirements, solver=self)
+            phase = _ResolvePhase(solver=self)
```

Comment:
This appears to be a bug:  `_ResolvePhase` only takes one argument.  mypy to the rescue.